.github/native-tests.json

```diff
@@ -105,7 +105,7 @@
         {
             "category": "Misc2",
             "timeout": 65,
-            "test-modules": "hibernate-validator, test-extension, logging-gelf, bootstrap-config, mailer, native-config-profile, locales",
+            "test-modules": "hibernate-validator, test-extension/tests, logging-gelf, bootstrap-config/application, mailer, native-config-profile, locales",
```

Comment:
Note: `test-extension/extension/deployment` does also contain tests, but those are not native-relevant (i.e. there are no executions in the non-incremental CI on `main` in that module).